app/Providers/AuthServiceProvider.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Providers;
+
+use App\Attendance;
+use App\Rsvp;
+use App\Policies\AttendancePolicy;
+use App\Policies\RsvpPolicy;
+use Illuminate\Support\Facades\Gate;
+use Illuminate\Foundation\Support\Providers\AuthServiceProvider as ServiceProvider;
+
+class AuthServiceProvider extends ServiceProvider
```

Comment:
Thoughts on renaming this to something more specific? In this app it will only ever be used for policies.

Reply:
I followed the Laravel suggestion for the name, but this could just be combined into the NovaServiceProvider if you'd prefer

Reply:
Hmmm. I think that would be cleaner.

Reply:
On second look that's not an option, it has to inherit from the AuthServiceProvider superclass.

Reply:
That's fine then